runtimes/runtimes/lsp/router/lspServer.ts

```diff
@@ -108,7 +108,10 @@ export class LspServer {
             this.lspConnection.console.log(
                 `Error in initialize handler: "${error}",\nwith initialization options: ${JSON.stringify(params.initializationOptions)}`
             )
-            return
+            return new ResponseError<InitializeError>(
+                ErrorCodes.InternalError,
+                error instanceof Error ? error.message : 'Unknown initialization error'
```

Comment:
Please decorate the returned error message with 
```
initialization options: ${JSON.stringify(params.initializationOptions)}`
```

This may seem repetitive, but what happens in line 108 and line 111 end up surfacing in different locations in the client, and users are more likely to give us the toolkit log, where this message is written. More importantly, we will receive this (line 111) string in our metrics, and this is the change I was initially asking for.